### DISCUSSION TITLE
STL files available on Thingiverse for a Nano-based Girs

### DISCUSSION THREAD

**Discussion by nothingTVatYT** [bengtmartensson/AGirs#70] [Show and tell]
I successfully build a Girs using the instructions and made a little case for easier handling meant to be 3d printed.
https://www.thingiverse.com/thing:5956632

**Comment by bengtmartensson** [comment]
Thank you very much for this. Sorry for not replying earlier. I am not really a 3D-printing person myself, though...

Is the given location "permanent"?

**Comment by nothingTVatYT** [comment]
Thingiverse kept the unique "thing id" since it launched as far as I know. I'm pretty sure this link will work permanently as there are so many links to Thingiverse out there.
